pages.de/common/poetry.md

```diff
@@ -0,0 +1,36 @@
+# poetry
+
+> Verwalte Python-Paketen und -Abhängigkeiten.
+> Weitere Informationen: <https://python-poetry.org/docs/cli/>.
+
+- Erstellen eins neuen Poetry-Projekts im Verzeichnis mit dem angegebenem Namen:
```

Comment:
I honestly hate the phrasing of the english page, but that's a discussion to be had somewhere else.

```suggestion
- Erstelle ein neues Poetry-Projekt im Verzeichnis mit dem angegebenem Namen:
```